src/logger/index.test.ts

```diff
@@ -16,114 +16,125 @@ const createTestLogger = (
   return { baseLogger, logger };
 };
 
-describe('log context', () => {
```

Comment:
It's not clear from the diff, but I only removed this describe block to make tests flat and the linter changed them from `it` to `test` which confused git.